src/frontend/js/index.tsx

```diff
@@ -91,7 +91,19 @@ document.addEventListener('DOMContentLoaded', async event => {
     try {
       translatedMessages = await import(`./translations/${locale}.json`);
     } catch (e) {
-      handle(e);
+      // Richie implicitly uses 'en-US' as its default locales: it does not provide localization files and strings
+      // for this locale but instead relies on them being the default messages throughout the code.
+      // We therefore do not want to report errors for a missing locale file that is not expected to exist in stock
+      // Richie. We still want errors if another locale is expected but missing, and we still want to log something
+      // to the console so Richie users who provide their own 'en-US' strings are warned if they are not loaded.
+      if (locale === 'en-US') {
+        // tslint:disable:no-console
+        console.log(
+          'No localization file found for default en-US locale, using default messages.',
```

Comment:
add a test if possible?

Reply:
Unfortunately I’m not sure how we could go about testing this since it’s just webpack internal machinery.

The index is not tested. I specifically left this code there when I refactored and made the root component due to the challenge of testing this